third_party_programs.txt

```diff
@@ -0,0 +1,1475 @@
+privately owned rights.
+
+=======================================================================================
+8. 	armon-go-metrics (MIT License)
+	https://github.com/armon/go-metrics
+	Copyright (c) 2013 Armon Dadgar
+
+	Dustin Sallings go-humanize (MIT License)
+	https://github.com/dustin/go-humanize
+	Copyright (c) 2005-2008  Dustin Sallings <dustin@spy.net>
+
+	Etcd-io bbolt (MIT License)
+	https://github.com/etcd-io/bbolt
+	Copyright (c) 2013 Ben Johnson
+
+	LibYAML	(MIT License)
+	http://pyyaml.org/wiki/LibYAML
+	Copyright (c) 2017-2020 Ingy döt Net Copyright (c) 2006-2016 Kirill Simonov
+
+	mitchellh-hashstructure	(MIT License)
+	http://github.com/mitchellh/hashstructure/
+	Copyright (c) 2016 Mitchell Hashimoto
+
+	Mockito	(MIT License)
+	https://github.com/mockito/mockito
+	Copyright (c) 2007 Mockito contributors
+
+
+--------------------------------------------------------------------
+	MIT License
+
+Permission is hereby granted, free of charge, to any person
+obtaining a copy of this software and associated documentation
+files (the "Software"), to deal in the Software without
+restriction, including without limitation the rights to use,
+copy, modify, merge, publish, distribute, sublicense, and/or
+sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT.  IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+=========================================================================
+
+9. 	libaio (GNU Lesser General Public License v2.1 or later)
+	https://pagure.io/libaio/tree/master
+	Copyright 2000,2001,2002 Red Hat, Inc.
+
+	Linux Libfuse (GNU Lesser General Public License v2.1 or later)
+	https://github.com/libfuse
+	https://github.com/libfuse/libfuse/blob/master/LICENSE
+	Copyright (C) 2001-2007  Miklos Szeredi <miklos@szeredi.hu>
+
+	Persistent Memory Programming ndctl (GNU Lesser General Public License v2.1 or later)
+	https://github.com/pmem/ndctl
+	https://github.com/pmem/ndctl/blob/master/LICENSES/preferred/LGPL-2.1
+	Copyright (C) 2015-2020 Intel Corporation. All rights reserved.
+
+------------------------------------------------------------------------------
+		 GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 2.1, February 1999
+
+ Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
```

Comment:
(style)  Do not include the paragraph about writing to the Free Software Foundation's mailing address from the sample GPL notice. The FSF has changed addresses in the past, and may do so again. Linux already includes a copy of the GPL.